schemas/EiffelMetaProperty/1.0.0.json

```diff
@@ -0,0 +1,147 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "meta": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "type": "string",
+        "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$"
+      },
+      "type": {
+        "type": "string",
+        "enum": [
+          "EiffelActivityCanceledEvent",
+          "EiffelActivityFinishedEvent",
+          "EiffelActivityStartedEvent",
+          "EiffelActivityTriggeredEvent",
+          "EiffelAnnouncementPublishedEvent",
+          "EiffelArtifactCreatedEvent",
+          "EiffelArtifactPublishedEvent",
+          "EiffelArtifactReusedEvent",
+          "EiffelCompositionDefinedEvent",
+          "EiffelConfidenceLevelModifiedEvent",
+          "EiffelEnvironmentDefinedEvent",
+          "EiffelFlowContextDefinedEvent",
+          "EiffelIssueDefinedEvent",
+          "EiffelIssueVerifiedEvent",
+          "EiffelSourceChangeCreatedEvent",
+          "EiffelSourceChangeSubmittedEvent",
+          "EiffelTestCaseCanceledEvent",
+          "EiffelTestCaseFinishedEvent",
+          "EiffelTestCaseStartedEvent",
+          "EiffelTestCaseTriggeredEvent",
+          "EiffelTestExecutionRecipeCollectionCreatedEvent",
+          "EiffelTestSuiteFinishedEvent",
+          "EiffelTestSuiteStartedEvent"
```

Comment:
To make the protocol extensible we shouldn't enumerate that valid event types here. A simple regexp pattern would do.

Reply:
Done

Reply:
I am not that sure when thinking about it. Please see my comment below.